src/PackageUtilities.js

```diff
@@ -141,8 +141,9 @@ export default class PackageUtilities {
     if (!Array.isArray(glob)) glob = [glob];
 
     const maybeNegate = negate ? (v) => !v : (v) => v;
+    const boolReducer = negate ? Array.prototype.every : Array.prototype.some;
 
-    return glob.some((glob) => maybeNegate(minimatch(pkg.name, glob)));
+    return boolReducer.call(glob, (glob) => maybeNegate(minimatch(pkg.name, glob)));
```

Comment:
Hmmm... this is getting a little roundabout.  Maybe simpler at this point to just have a top-level if/else?

Like:

```js
if (negate) {
  return glob.every((glob) => !minimatch(pkg.name, glob));
} else {
  return glob.some((glob) => minimatch(pkg.name, glob));
}
```